src/models/exhibitor.model.ts

```diff
@@ -0,0 +1,28 @@
+import { model, Schema } from 'mongoose';
+
+import type { IExhibitor } from '../types/exhibitor.type';
+
+const exhibitorSchema: Schema = new Schema(
+  {
+    name: {
+      type: String,
+      required: true,
+    },
+    picture: {
+      type: String,
+    },
+    place: {
+      type: String,
+    },
+    sectors: {
+      type: Array<String>,
+    },
+    interests: {
+      type: Array<String>,
+    }
```

Comment:
Pour info, pour gagner en place + lisibilité tu peux faire :
```suggestion
    picture: String,
    place: String,
    sectors: String[],
    interests: String[]
```